api/src/main/resources/workbench.yaml

```diff
@@ -277,6 +277,26 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /v1/page-visits:
+    post:
+      tags:
+      - profile
+      description: Updates a users page visits
+      operationId: updatePageVisits
+      parameters:
+        - in: body
+          name: pageVisit
+          description: the users pageVisits
+          schema:
+            $ref: "#/definitions/PageVisit"
+      responses:
+        204:
```

Comment:
It is likely we will want this to return either the page visits, or the entire profile (the latter would be consistent with the TOS/demographics/idv APIs). This would be a requirement for the profile storage work I mentioned in another comment. Feel free to do that now if it's easy, otherwise it can wait.